runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java

```diff
@@ -286,14 +286,16 @@ public enum ItemMapping
 	ITEM_INFINITY_HAT_DARK_COLOUR_KIT(DARK_INFINITY_COLOUR_KIT, DARK_INFINITY_HAT),
 
 	// Miscellaneous ornament kits
-	ITEM_DARK_BOW(DARK_BOW, DARK_BOW_12765, DARK_BOW_12766, DARK_BOW_12767, DARK_BOW_12768, DARK_BOW_20408, DARK_BOW_BH),
+	ITEM_DARK_BOW(DARK_BOW, DARK_BOW_12765, DARK_BOW_12766, DARK_BOW_12767, DARK_BOW_12768, DARK_BOW_20408, DARK_BOW_BH, DARK_BOW_DEADMAN),
 	ITEM_ODIUM_WARD(ODIUM_WARD, ODIUM_WARD_12807),
 	ITEM_MALEDICTION_WARD(MALEDICTION_WARD, MALEDICTION_WARD_12806),
 	ITEM_STEAM_BATTLESTAFF(STEAM_BATTLESTAFF, STEAM_BATTLESTAFF_12795),
 	ITEM_LAVA_BATTLESTAFF(LAVA_BATTLESTAFF, LAVA_BATTLESTAFF_21198),
 	ITEM_TZHAARKETOM(TZHAARKETOM, TZHAARKETOM_T),
 	ITEM_TZHAARKETOM_ORNAMENT_KIT(TZHAARKETOM_ORNAMENT_KIT, TZHAARKETOM_T),
 	ITEM_DRAGON_HUNTER_CROSSBOW(DRAGON_HUNTER_CROSSBOW, DRAGON_HUNTER_CROSSBOW_B, DRAGON_HUNTER_CROSSBOW_T),
+	ITEM_VOIDWAKER(VOIDWAKER, VOIDWAKER_DEADMAN),
+	ITEM_VOLATILE_NIGHTMARE_STAFF(VOLATILE_NIGHTMARE_STAFF, VOLATILE_NIGHTMARE_STAFF_DEADMAN),
```

Comment:
Does this actually work? I would expect this to need to be added instead to the `ITEM_VOLATILE_ORB` and `ITEM_NIGHTMARE_STAFF` entries.

Reply:
Good callout, moved it to those items.